backend/seed.py

```
@@ -161,6 +162,38 @@ def load_hints():
+        module = module_title_to_module.get(module_data["module_title"])
+        if module:
+            module.target_runtime = module_data.get("target_runtime")
+            for test_case in module_data["test_cases"]:
+                test_case_instance = TestCase(
+                    module_id=module.id,
+                    input=test_case["input"],
+                    output=test_case["output"],
+                )
+                test_cases_to_add.append(test_case_instance)
+        else:
+            logger.error(f"Module with title {module_data['module_title']} not found.")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)